src/style.css

```diff
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+  font-size: 16px;
+}
+
+/* body {
+  margin: 20px 10px 0;
+} */
+
+.form {
+  /* display: inline-block; */
+  width: 100%;
+  margin-top: 20px;
+  background: #fff;
+}
+
+.bigInput {
+  display: inline-block;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  /* padding: 24px 0;
+  line-height: 20px; */
+}
+
+.bigInput::placeholder {
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+}
+
+.bigImage {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
```

Comment:
you are using the exact same styles for two search bars. let's create a class `searchBar` or anything you like, that will have all the common styles for two searchbars. then create two modifiers of that class, for example `searchBar--big` and `searchBar--small`. these will have everything that is different. you can also remove separate classes for images and move everything under that one `searchBar` class and the modifiers. it will be much easier to work with